### DISCUSSION TITLE
Resource Access Option - Missing documentation of new option names

### DISCUSSION THREAD

**Discussion by a82kd** [sandboxie-plus/Sandboxie#1805] [Documents and notes]
### Is your feature request related to a problem or use case?

The access options have been renamed in more recent versions (Normal, Open, Open for All, Closed, Read Only, Box Only).
However, the meaning of these options in not alway clear (e.g. Normal vs. Open?)

### Describe the solution you'd like

Please add either a description of options (tooltip?) in the software. Or alternatively please update the documentation.

**Comment by DavidXanatos** [comment]
Normal is only relevant when usign rule specificity, 
normal is the norlam behavioure of sandboxie, reading alowed but writes are virtualized.
setting a path to normal is only menainfgull when a parent path was first set to soemthign else, like done in the privacy mode.

**Comment by a82kd** [comment]
This is true for the regular sandboxes. However, if I understand your answer correctly, for a **Privacy Enhanced** sandbox the **Normal** option would be the preferred option to open directories, right?

**Comment by DavidXanatos** [comment]
yes then you allow read but still prevent write

**Comment by lwcorp** [comment]
Could you add tooltips for all these options?
Also, why is the last one cut off?
![image](https://user-images.githubusercontent.com/1773306/171233678-f79c0b0b-ae28-4e0f-aa01-d06bc9bbefa8.png)

**Reply by DavidXanatos** [reply]
I have to check if QT allows tooltips for combo box items, its cut of because the size is made to fit the column you cna resize the column to fit it all

**Reply by lwcorp** [reply]
Thanks! In the mean time can you add those descriptions here or to a help page (and link to it here)?

**Reply by JerryZhongJ** [reply]
I have the same question, what is the difference between open and open for all? And how could I make a sub-folder open (readable and writable) while keep all the other files and folders in the parent folder invisible (not readable or writable).

**Reply by isaak654** [reply]
Open = [OpenFilePath](https://sandboxie-plus.github.io/sandboxie-docs/Content/OpenFilePath.html)
Open for All = [OpenPipePath](https://sandboxie-plus.github.io/sandboxie-docs/Content/OpenPipePath.html)

Besides, [this table](https://sandboxie-plus.github.io/sandboxie-docs/Content/ResourceAccess.html) should list all resource access settings. 

Any further contribution to the [Sandboxie-docs](https://github.com/sandboxie-plus/sandboxie-docs) repository would be helpful.
You may also want to check all issues tagged with the [documentation](https://github.com/sandboxie-plus/Sandboxie/issues?q=label%3Adocumentation) label or the [discussions](https://github.com/sandboxie-plus/Sandboxie/discussions/categories/documentation-and-notes) too.

**Reply by JerryZhongJ** [reply]
Thanks, it's a great help.

**Reply by lwcorp** [reply]
I also see the last option is no longer cut off, thanks for fixing it!
Although it's not clear if your link explains this last option so can you explain it here?
Keep in mind all these options apply only to a certain box.

![image](https://user-images.githubusercontent.com/1773306/193833144-924566f7-fc87-4b17-aacd-5081559b3f86.png)

**Reply by isaak654** [reply]
If you apply the `Box Only (Write Only)` rule with a dummy path and then go to the `Edit ini Section` tab shown in the screen above, you will see the applied parameter (in this case `WriteFilePath`) which can be found [here](https://sandboxie-plus.github.io/sandboxie-docs/Content/ResourceAccess.html) or in the GitHub search in this repository.
